Doc/whatsnew/3.10.rst

```diff
@@ -885,6 +885,10 @@ Other Language Changes
 
 *  A :exc:`SyntaxError` (instead of a :exc:`NameError`) will be raised when deleting the :const:`__debug__` constant. (Contributed by Dong-hee Na in :issue:`45000`.)
```

Comment:
```suggestion
*  A :exc:`SyntaxError` (instead of a :exc:`NameError`) will be raised when deleting
   the :const:`__debug__` constant.  (Contributed by Dong-hee Na in :issue:`45000`.)
```